projects/uitdatabank/reference/entry.json

```diff
@@ -2750,6 +2749,63 @@
         ]
       }
     },
+    "/events/{eventId}/permissions": {
+      "parameters": [
+        {
+          "$ref": "#/components/parameters/eventId"
+        }
+      ],
+      "get": {
+        "summary": "events - permissions",
```

Comment:
I think the summary should be "permissions - get"

<img width="263" alt="Screenshot 2025-02-07 at 08 42 47" src="https://github.com/user-attachments/assets/b0346c1f-fb8c-415b-ab5f-3d41a0aeaa8b" />